autopush/logging.py

```diff
@@ -113,12 +113,17 @@ def __call__(self, event):
 
     def raven_log(self, event):
         f = event["log_failure"]
-        stack = extra = None
+        stack = None
```

Comment:
Addressed in later patch. The "stack" element that was causing issues was removed. "log_namespace" specifies the module where the exception occurred, and added a "uri" element to client_info to specify the request path. These should provide enough info to determine the source of the error.